test/integration/master/kms_transformation_test.go

```diff
@@ -197,23 +198,27 @@ resources:
        endpoint: unix:///@kms-provider-2.sock
 `
 
-	pluginMock1, err := newBase64Plugin("@kms-provider-1.sock")
+	pluginMock1, err := mock.NewBase64Plugin("@kms-provider-1.sock")
 	if err != nil {
 		t.Fatalf("failed to create mock of KMS Plugin #1: %v", err)
 	}
 
-	go pluginMock1.grpcServer.Serve(pluginMock1.listener)
-	defer pluginMock1.cleanUp()
-	kmsPluginMustBeUp(t, pluginMock1)
+	go pluginMock1.Start()
```

Comment:
Start() launches a goroutine internally, so we can drop the `go` bit here, right? (applies to all callers)